aasemble/django/apps/buildsvc/views.py

```diff
@@ -92,6 +92,17 @@ def builds(request):
 
 
 @login_required
+def rebuild(request, source_id):
+    ps = PackageSource.objects.get(pk=source_id)
+    try:
+        ps.build()
+    except Exception as e:
+        ps.source_id = 'exception'
+        print(e)
+    return render(request, 'buildsvc/html/rebuild.html', {'source': ps})
```

Comment:
I'm not entirely sure what's going on here.

Printing the exception is of no use. The end user will never see it. It'll land in a web server log somewhere.

Why set the source_id to 'exception'?